examples/linear_deformation.py

```diff
@@ -0,0 +1,1187 @@
+# Copyright 2014-2016 The ODL development group
```

Comment:
Agree that pushing it under `discr` could cause it to become bloated. Perhaps another top level package is the best way to go for it, at least if we're going to add several of these.